docs/authenticators.md

```
@@ -0,0 +1,85 @@
+# Custom Authenticators (LDAP, etc.)
+
+Some Cassandra clusters authenticate clients with a non-standard authenticator —
+for example `org.apache.cassandra.auth.LDAPAuthenticator`. By default the driver
+used by this plugin (gocql) only accepts a fixed allow-list of server-side
+authenticator class names, and refuses to connect to anything else with an error
+like:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)